src/dmd/mars.d

```diff
@@ -1361,7 +1361,37 @@ void addDefaultVersionIdentifiers()
     static if (TARGET.Windows)
     {
         if (global.params.mscoff)
+        {
             VersionCondition.addPredefinedGlobalIdent("CRuntime_Microsoft");
+            if (!global.params.mscrtlib || !global.params.mscrtlib[0])
+            {
+                VersionCondition.addPredefinedGlobalIdent("MSRuntime_None");
+            }
+            else if (!strcmp(global.params.mscrtlib, "libcmt"))
```

Comment:
Windows has a case insensitive file system, so `strcmp` is unreliable. Use `FileName.compare`.

Reply:
`FileName.equals`?

Reply:
that works too